src/main/java/net/mindengine/galen/reports/GalenTestAggregatedInfo.java

```diff
@@ -76,21 +100,59 @@ public String getExceptionStacktrace() {
         }
         return null;
     }
-    
+
     public Long getDuration() {
         return testInfo.getEndedAt().getTime() - testInfo.getStartedAt().getTime();
     }
-    
+
+    public Set<String> getIncludedTags() {
+        return includedTags;
+    }
+
+    public void setIncludedTags(final Set<String> includedTags) {
+        this.includedTags = includedTags;
+    }
+
+    public Set<String> getExcludedTags() {
+        return excludedTags;
+    }
+
+    public void setExcludedTags(final Set<String> excludedTags) {
+        this.excludedTags = excludedTags;
+    }
+
+    public String getTagsPretty() {
```

Comment:
Could you please merge your changes with ft-119-testgroups branch. The thing is I have already merged you previous pull request into separate branch and in there I have removed this method together with the code that collects tags from layout reports. But the thing is that functionality of test groups is not yet completely finished.